sdk/js/src/Client.ts

```diff
@@ -246,6 +248,10 @@ export class DVCClient implements Client {
             document.removeEventListener('visibilitychange', this.pageVisibilityHandler)
         }
 
+        if (typeof window !== 'undefined' && this.windowMessageHandler) {
```

Comment:
does this need the react native check as well? You could also just only assign the windowMessageHandler property if its being registered above, then just check if it exists here

Reply:
yeah, I'll update so I only assign the `windodwMessageHandler` when we add the event handler